test/unit/CowOperator/setup.js

```diff
@@ -46,12 +46,14 @@ async function setup() {
   await daiAggregator.setLatestAnswer(0.0002 * 1e18); // 1 dai = 0.0002 eth, 1 eth = 5000 dai
   const stethAggregator = await ChainlinkAggregatorMock.deploy();
   await stethAggregator.setLatestAnswer(parseEther('1')); // 1 steth = 1 eth
+  const usdcAggregator = await ChainlinkAggregatorMock.deploy();
+  await usdcAggregator.setLatestAnswer(0.0002 * 1e18); // 1 usdc = 0.0002 eth, 1 eth = 5000 dai
```

Comment:
Although you have changed the decimals below, the exchange rate set here still uses 18 decimals. Shouldn't have this failed some tests?